app/javascript/src/members/member_payments.js

```diff
@@ -0,0 +1,44 @@
+$(document).on('ready page:load turbolinks:load', function() {
```

Comment:
As far as I can see we use 2 instead of 4 spaces as indentation. Please try to keep that the same everywhere.